pandas/tests/frame/test_analytics.py

```diff
@@ -249,6 +249,19 @@ def test_bool_describe_in_mixed_frame(self):
                              index=['count', 'unique', 'top', 'freq'])
         tm.assert_frame_equal(result, expected)
 
+    def test_describe_only_bool_frame(self):
```

Comment:
There are now tests for all the `DataFrame` cases in [`frame/test_analytics.py`](https://github.com/agraboso/pandas/blob/aa1b96de5cd871b6d11e3151ee3e13bc6b11dcf2/pandas/tests/frame/test_analytics.py).

I could not find any tests for `Series.describe()`, so I added some to [`series/test_analytics.py`](https://github.com/agraboso/pandas/blob/aa1b96de5cd871b6d11e3151ee3e13bc6b11dcf2/pandas/tests/series/test_analytics.py). According to [generic.py#L5128-L5136](https://github.com/agraboso/pandas/blob/aa1b96de5cd871b6d11e3151ee3e13bc6b11dcf2/pandas/core/generic.py#L5128-L5136), there are four different cases, depending on the `dtype` of the `Series`: boolean, numeric, `timedelta64` and other. The tests I added go over three of these &mdash; all except `timedelta64`. How do I create a meaningful `timedelta64` series?